src/components/MyRockets.js

```diff
@@ -1,11 +1,38 @@
-import React from 'react';
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { loadRockets } from '../redux/rockets/rockets';
+import '../stylesheets/rockets.css';
 
 function MyRockets() {
+  const dispatch = useDispatch();
+  const rockets = useSelector((state) => state.rocketsReducer);
```

Comment:
You have done a good job, Rida. this script should be in the `rocket.js` file. MyRockets component is for the reserved rockets.